packages/opentelemetry-instrumentation-xml-http-request/src/xhr.ts

```diff
@@ -46,6 +46,16 @@ import { AttributeNames } from './enums/AttributeNames';
 // safe enough
 const OBSERVER_WAIT_TIME_MS = 300;
 
+// Used to normalize relative URLs
+let a: HTMLAnchorElement | undefined;
```

Comment:
web already has this function, no need to copy this here again

Reply:
Ok I took it from fetch(#2010) but you are right we should reuse it. Issue is that getUrlNormalizingAnchor is not exported in utils right now. I added the export in this PR and then after release replace the xhr/fetch copy pasted ones with it.

Reply:
hmm I'm not sure if I understand. It looks like you added an export to this function in web but you still haven't use that ? Why it cannot be done now but after release ?

Reply:
nvm I misunderstood something, fixed.